dali/operators/reader/loader/numpy_loader.h

```diff
@@ -52,13 +51,14 @@ class NumpyParseTarget{
   }
 };
 
+DLL_PUBLIC void ParseHeaderMetadata(NumpyParseTarget& target, const std::string &header);
```

Comment:
```suggestion
void ParseHeaderMetadata(NumpyParseTarget& target, const std::string &header);
```
I don't think we should expose this function.

Reply:
You wouldn't be able to test it otherwise...

Reply:
we do if we want to test it